GitHome6/Patient.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GitHome6
+{
+    public class Patient : Person
+    {
+
```

Comment:
Please, remove redundant blank spaces between class name and method name

Reply:
done